easybuild/easyconfigs/v/VariationHunter/VariationHunter-0.4-foss-2016b.eb

```diff
@@ -0,0 +1,35 @@
+#  easybuild easyconfig
+#
+#  John Dey  jfdey@fredhutch.org
+#
+#  Fred Hutchinson Caner Research Center
+#
+#  Note: distribution uses version in an inconsistent fasion: 0.04 and 0.4 
+
+easyblock = 'MakeCp'
+
+name = 'VariationHunter'
+version = '0.4'
+
+homepage = 'http://variationhunter.sourceforge.net/Home'
+description = """Variation Hunter - CommonLaw is a tool for discovery of structural variation in one or more individuals simultaneously using high throughput technologies."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [('https://sourceforge.net/projects/%(namelower)s/files/V%(version)s/', 'download')]
+sources = ['%(name)s_CL.v0.04.tar.gz']
```

Comment:
@fizwit please don't hardcode the version in two places?

use `%(version)s` instead?

or is the tarball for v0.4 really called `...0.04.tar.gz`?